rpm/README.md

```diff
@@ -27,18 +27,18 @@ and enhancements that were not available until after the software release.
 
 2. Download and upgrade the latest documentation RPM.
 
-   For example, to download and upgrade to the latest SAT 2.4 documentation RPM:
+   For example, to download and upgrade to the latest SAT documentation RPM:
```

Comment:
It might be worth discussing this a bit further, if you are interested. I had Mikhail change the layout of the release repository so that the RPMs are all in the same directory, thinking that would be easier so we don't have to update version numbers.

However, this has a slight issue that there will only be one docs-sat-latest.noarch.rpm now whereas previously there was one "docs-sat-latest.noarch.rpm" for each major/minor version of SAT, e.g. a latest 2.4, a latest 2.5, etc.

In general I think the flatter layout is a bit nicer, but it is possible we would have issues with the "latest" link.

Reply:
@ekamin-hpe, I'm approving the changes in this PR, but I'd be interested in discussing the layout of the release repository more in our next meeting.

Reply:
I filed https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1675 for this. We decided this would be okay to revisit when there actually are multiple major/minor versions.